cogs5e/sheets/gsheet.py

```diff
@@ -265,6 +266,9 @@ def _gchar(self):
         elif vcell:
             self.additional = TempCharacter(doc.worksheet('Additional'))
             self.version = (2, 1) if "2.1" in vcell else (2, 0) if "2" in vcell else (1, 0)
+            if self.version >= (2, 0):
+                self.inventory = TempCharacter(doc.worksheet('Inventory'))
```

Comment:
Was this added in GSheet v2.0? We should make sure this is not a backward-incompatible change.
cc @Croebh @thecoppersanctum